charm/kafka/lib/charms/layer/kafka.py

```diff
@@ -1,57 +1,60 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Let's leave the license attribution in. This file is a derivative work of an Apache bigtop charm...